public/examples/integrations/toolkits/slack/get_conversation_history_by_id_example_call_tool.js

```diff
@@ -0,0 +1,34 @@
+import { Arcade } from "@arcadeai/arcadejs";
+
+const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable
+
+const USER_ID = "you@example.com";
+const TOOL_NAME = "Slack.GetConversationHistoryById";
+
+// Start the authorization process
+const authResponse = await client.tools.authorize({
+    tool_name: TOOL_NAME,
+    user_id: USER_ID,
+});
+
+if (authResponse.status !== "completed") {
```

Comment:
Thinking out loud: I wonder if adding a `status.isComplete()` helper would be nice here

Reply:
This is not for now, I'll file a ticket. We should think about how clients should handle errors here.